mlflow/pytorch/__init__.py

```diff
@@ -539,14 +630,39 @@ def load_model(model_uri, **kwargs):
     .. code-block:: python
         :caption: Example
 
+        import numpy as np
         import torch
-        import mlflow
         import mlflow.pytorch
-        # Set values
-        model_path_dir = ...
-        run_id = "96771d893a5e46159d9f3b49bf9013e2"
-        pytorch_model = mlflow.pytorch.load_model("runs:/" + run_id + "/" + model_path_dir)
-        y_pred = pytorch_model(x_new_data)
+
+        # Class defined here
+        class LinearNNModel(torch.nn.Module):
+            ...
+
+        # Initialize our model, criterion and optimizer
+        model = LinearNNModel()
+        ...
+
+        # Training loop
+        ...
+
+        # Log the model
+        with mlflow.start_run() as run:
+            mlflow.pytorch.log_model(model, "models_pth")
+
+        # Inference after loading the logged model
+        model_uri = "runs:/{}/models_pth".format(run.info.run_id)
+        loaded_model = mlflow.pytorch.load_model(model_uri)
+        for hv in [4.0, 6.0, 30.0]:
```

Comment:
Can we pass `[4.0, 6.0, 30.0]` to the model as a tensor?

```
model(xxx_yyy([4.0, 6.0, 30.0]))
```

Reply:
You can do something like this since the network expects the input as shape (n, 1) tensor.

```
X = torch.tensor([4, 6, 30]).view(-1,1).float()
y_pred = loaded_model(X)
print(X)
print(y_pred)
```

But I prefer to simplify and send individual tensor so we can print the X, --> y on the same line. Yes, for larger models
you want your input to be the whole batch, but for here I just made it simple: inferring one tensor at a time.